server/lib/poller.js

```diff
@@ -65,29 +65,61 @@ class Poller
         }
 
         try {
-
             // Check accounts and operations!
-            let checkAccounts = await Config.findOrCreateDefaultBooleanValue(
+            let checkAccounts = false;
```

Comment:
checkAccounts is defined out of the try. Do you mean first try? It is not necessary to be in this scope, no?